README.md

```diff
@@ -26,10 +32,16 @@ These instructions will get you a copy of the project up and running on your loc
 1. Run the installation script. `npm install` (If you see `babel-node: command not found`, you've missed this step.)
 1. Generate Canvas API token and copy it to clipboard.
 1. Create a `.env` file.
-2. Add the following: `CANVAS_API_TOKEN={YOUR API TOKEN}` and `CANVAS_API_DOMAIN={YOUR API DOMAIN}`. An example `CANVAS_API_DOMAIN` is `https://{school}.instructure.com/api/v1`
-3. Add your course ID to `index.js`, where it says: `//{course id} add course ID here!` 
-4. Run the script. `npm start`.
-5. An `output.csv` file should be generated with discussion data in the output folder.
+1. Add the following: `CANVAS_API_TOKEN={YOUR API TOKEN}`, `CANVAS_API_DOMAIN={YOUR API DOMAIN}`, `COURSE_IDS={YOUR COURSE ID(s)}`. At UBC the `CANVAS_API_DOMAIN` is `https://{school}.instructure.com/api/v1`
```

Comment:
Since we're already specifying that it's "At UBC", we could change the link to `https://ubc.instructure.com/api/v1` rather than `https://{school}.instructure.com/api/v1`